diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -159,9 +159,28 @@ controller_interface::return_type DiffDriveController::update(
     double right_feedback_mean = 0.0;
     for (size_t index = 0; index < static_cast<size_t>(wheels_per_side_); ++index)
     {
-      const double left_feedback = registered_left_wheel_handles_[index].feedback.get().get_value();
-      const double right_feedback =
-        registered_right_wheel_handles_[index].feedback.get().get_value();
+      bool left_success = false;
```

Comment:
shall we simplify this and use just a single bool variable if we don't use the information of left/right failure?
Please also add a comment here, that we only need the lambda functions here to be able to mark the double left/right_feedback variable as const.

Reply:
```cpp
  bool success = true;
    // lambda function only to mark `double left_feedback` as `const`.
    const double left_feedback = [&]()
    {
      double temp{};
      success = registered_left_wheel_handles_[index].feedback.get().get_value(temp);
      return temp;
    }();

    // lambda function only to mark `double right_feedback` as `const`.
    const double right_feedback = [&]()
    {
      double temp{};
      success = registered_right_wheel_handles_[index].feedback.get().get_value(temp);
      return temp;
    }();

    if (!success)
    {
      RCLCPP_WARN(
        get_node()->get_logger(),
        "Error while fetching information from the state interfaces for %s wheel at index [%zu]",
        (index % 2 == 0) ? "left" : "right", index);
      return controller_interface::return_type::OK;
    }

```

@christophfroehlich  Is this make sense ?

Reply:
while working on another controller, I noticed that I often need to implement a similar setup for `get_value()`across multiple places, wouldn't it be a good idea to create a separate reusable function for this? We could then import it wherever needed, which might also be helpful in test files.

what do you think?